src/locale/languages/ptBr.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
   'Are you sure?': 'Tem certeza?',
   'Assign Yourself': 'Atribua a você',
   Assignees: 'Atribuída a',
-  'Author: ': 'Autor',
+  'Author: ': 'Autor: ',
```

Comment:
Yes, this should be the root case. Can you do me a favor to add similar colon and space for 'Author: ' in it/pt/sv and 'Committer: ' in sv?

Reply:
Sure